src/v2/Apps/Home/HomeApp.tsx

```diff
@@ -0,0 +1,58 @@
+import { Column, GridColumns, Spacer } from "@artsy/palette"
+import { compact } from "lodash"
+import React from "react"
+import { Title } from "react-head"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useSystemContext } from "v2/System/useSystemContext"
+import { HomeApp_homePage } from "v2/__generated__/HomeApp_homepage.graphql"
+import { HomeHeroUnitFragmentContainer } from "./Components/HomeHeroUnit"
+import { HomeInfoBlurb } from "./Components/HomeInfoBlurb"
+
+interface HomeAppProps {
+  homePage: HomeApp_homePage
+}
+
+export const HomeApp: React.FC<HomeAppProps> = ({ homePage }) => {
+  const heroUnits = compact(homePage.heroUnits?.slice(0, 2)) ?? []
```

Comment:
We definitely consistently have more than 2 of these — am curious what we want to do with them @deliciousnachos

Reply:
Ah yeah I need to give you a spec for that - will do